config/environments/development.rb

```
@@ -5,9 +5,19 @@
   # Otherwise use Letter Opener to preview emails in the browser.
   if ENV["SMTP_HOST"].present?
     config.action_mailer.delivery_method = :smtp
-    config.action_mailer.smtp_settings = { address: ENV["SMTP_HOST"], port: ENV.fetch("SMTP_PORT", 1025).to_i }
+    config.action_mailer.smtp_settings = {
+      address:              ENV["SMTP_HOST"],
```

RuboCop (1.86.2)

[convention] Align the keys of a hash literal if they span more than one line.

(Layout/HashAlignment)